pkg/gatherers/common/gather_logs.go

```diff
@@ -116,8 +116,7 @@ func CollectLogsFromContainers( //nolint:gocyclo
 			}
 
 			if containersFilter.MaxNamespaceContainers > 0 && len(records) >= containersFilter.MaxNamespaceContainers {
-				errs = append(errs, fmt.Errorf("skipping %s for %s. Max containers per namespace reached (max: %d)",
-					containerName, containersFilter.Namespace, containersFilter.MaxNamespaceContainers))
+				skipped++
```

Comment:
Isn't it `len(pods.Items) - containersFilter.MaxNamespaceContainers`?

Reply:
It also works.

Reply:
Yeh, maybe half correct?

``len(containerNames) - containersFilter.MaxNamespaceContainers``

Reply:
Unless we want to get the number of pods, not the containers... which is fine as well...

Reply:
Nah, nope. We need to know the number of containers, 'cause that is what we are limiting. So, counting the `pod.Items` isn't what we want.

Reply:
Also, 'cause we have some regex rule that checks if the containerName matches some rules at L#94, well... it can cause some misleading.

Reply:
I kind of used a similar logic to count the `containerNames` as well to break the loop when the limit reaches, so... it will avoid entering in some unnecessary for to found stuff.